### PR TITLE
Gallery

### PR DESCRIPTION
**Pull request by jukent**
Some work to create a separate links gallery. See #44.

Resolved (thanks @andersy005):
This PR currently should not pass the tests as I can't build it locally either. I have made some progress but am stuck at the error 
```
Extension error:
Handler <function main at 0x111735700> for event 'builder-inited' threw an exception (exception: name 'dedent' is not defined)
make: *** [html] Error 2
```
and am having a hard time tracking it down. I see where `dedent` is called, but cannot see the difference between its call in `yaml_gallery_generator.py` and in `conf.py`. I am also wondering if the numbers in the error are significant.

### CODE REVIEW

**Comment by andersy005** [comment]
@jukent, you are missing an import:

```python
from textwrap import dedent
```

**Comment by jukent** [comment]
> @jukent, you are missing an import:
> 
> ```python
> from textwrap import dedent
> ```

Thank you so much!

**Comment by andersy005** [comment]
> Thank you so much!

Of course!

**Comment by jukent** [comment]
To Do:

Make tags clickable / searchable
Make description collapsed with option to expand it
Add images for each links.yaml entry
Use more author information

**Comment by jukent** [comment]
@ProjectPythia/education Please take a look at this and let me know if you'd like any descriptions, author information, or categories changed for a given entry. Any other comments are appreciated.

**Comment by clyne** [comment]
> @ProjectPythia/education Please take a look at this and let me know if you'd like any descriptions, author information, or categories changed for a given entry. Any other comments are appreciated.

What is the best way to view this information? I don't seem to be able to build locally following our contributor's guide and not sure where to look if I'm successful :-)

**Comment by jukent** [comment]
@clyne I can build it locally with 
```
make html
open _build/html/index.html
```

Is that not working for you? Alternatively you can click "Show all checks" next to the green "All checks have passed". Then select "Details" and you will see a test build of the site.

**Comment by clyne** [comment]
> @clyne I can build it locally with
> 
> ```
> make html
> open _build/html/index.html
> ```
> 
> Is that not working for you? Alternatively you can click "Show all checks" next to the green "All checks have passed". Then select "Details" and you will see a test build of the site.

My bad. I wasn't in the content directory. Thanks.

**Comment by clyne** [comment]
> @ProjectPythia/education Please take a look at this and let me know if you'd like any descriptions, author information, or categories changed for a given entry. Any other comments are appreciated.

I think it would be helpful to try to come up with an initial set of classifiers to choose from for each of the entries. In the future external contributors could be provided a list to choose from as appropriate. Here is a first stab at sets and elements to consider:

Packages: numpy, matplotlib, xarray, dask, metpy, geocat, etc.
Formats: video, gallery, book, course
Level: beginner, intermediate, advanced
Science topics: climate, weather, ocean, etc.
Technologies: Python, Conda, GitHub, etc.

Thoughts?

**Comment by jukent** [comment]
At last week's meeting we decided we'd merge this as everyone who spoke up felt it was an improvement over the current table. For some reason I am struggling to request review from @ProjectPythia/infrastructure

**Comment by brian-rose** [comment]
This is all new to me, but GitHub offered me a button called "ready for review". Clicking it seems to have triggered the review requests based on who's listed as CODEOWNERS.

I'm not sure if we have the teams enabled for this repo? @andersy005 do we need to take some action to active that?

**Review by brian-rose — CHANGES_REQUESTED** [review]
I just sharpened the description of my Climate Laboratory book.

Everything else looks fine to me at first glance. I hope we can merge this and tweak details in future PRs, especially as we get more eyes on these pages. E.g. the authors of some of the external resource might be interested in contributing an updated description.

**Comment by dopplershift** [comment]
@brian-rose For future reference, clicking "ready for review" converts the PR from a "Draft" to one that's can be merged/considered ready for review.

File: content/links.yaml [hunk 1]
```diff
@@ -0,0 +1,642 @@
...
+    - name: Kerry Key
+      affiliation: Lamont-Doherty Earth Observatory
+  thumbnail: ../_static/thumbnails/earth_env_data_science.png
+  tags:
+    - course
+
+- title: The Climate Laboratory
+  url: https://brian-rose.github.io/ClimateLaboratoryBook/home
+  description: |
+    A hands on approach to climate physics and climate modeling. This JupyterBook serves as a course textbook at the University of Albany. It is self-reproducing, free and open, interactive, and is a living document.
```
[the remark this answers is not in the document]
Comment (jukent) [reply content/links.yaml]: Made this change in the most recent push.

**Review by brian-rose — APPROVED** [review]

**Review by mgrover1 — APPROVED** [review]
This looks great!!

**Pull request merged**